src/modules/assignment/assignment.controller.ts

```diff
@@ -88,9 +87,14 @@ export class AssignmentController {
   }
 
   // Delete a Assignment
-  @Delete('/delete')
-  async deleteAssignment(@Res() res, @Query('uid') uid) {
-    const Assignment = await this.AssignmentService.deleteAssignment(uid);
+  @Delete('/:AssignmentId')
+  async deleteAssignment(
+    @Res() res,
+    @Param('AssignmentId') AssignmentId: string,
+  ) {
+    const Assignment = await this.AssignmentService.deleteAssignment(
+      AssignmentId,
+    );
```

Comment:
It can be written in one line

Reply:
Linting makes this appear in a new line (npm run lint:fix)